NoNicotine_Business/Handler/GetPatientQueryHandler.cs

```diff
@@ -1,6 +1,7 @@
 ﻿using MediatR;
 using Microsoft.Extensions.Logging;
 using NoNicotine_Business.Queries;
+using NoNicotine_Business.Repositories;
```

Comment:
Changes exclusively for testing purposes shouldn't be merged with the master branch